### PR TITLE
[doc][kerberos] add kerberos doc for hive catalog

### PR DESCRIPTION
**Pull request by wsjz**
add doc for https://github.com/apache/doris/pull/36430

### CODE REVIEW

**Comment by shasevendow** [comment]
1. Please also update the EN doc.
2. Is the doc also applicable to the 2.1 version? If so, update the 2.1 version docs.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md [hunk 1]
```diff
@@ -547,6 +547,77 @@ CREATE CATALOG hive_krb_ha PROPERTIES (
 );
 ```
 
+### 支持同时访问第二个Kerberos集群
```
Comment (morningman) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md:550]:
```suggestion
### 多 Kerberos 集群配置
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md [hunk 2]
```diff
@@ -547,6 +547,77 @@ CREATE CATALOG hive_krb_ha PROPERTIES (
...
+
+如需同时访问另一个启用Kerberos的Hadoop集群，需要修改krb5.conf文件并且配置`hadoop.security.auth_to_local`属性，具体操作如下：
```
Comment (morningman) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md:552]:
```suggestion
如需同时访问多个启用了Kerberos的Hadoop集群，需要修改krb5.conf文件并且配置`hadoop.security.auth_to_local`属性，具体操作如下：
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md [hunk 3]
```diff
@@ -547,6 +547,77 @@ CREATE CATALOG hive_krb_ha PROPERTIES (
...
+
+1. 在krb5.conf文件配置realms
+
+配置多集群时，需要把把多个realm配置到一个krb5.conf里头，kdc和admin_server也可以是域名。
```
Comment (morningman) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md:556]:
```suggestion
配置多集群时，需要把多个realm配置到一个krb5.conf里头，kdc和admin_server也可以是域名。
```

Comment (morningman) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md:556]: 注意，这部分要缩进一格，因为在 `1. 在krb5.conf文件配置realms` 标题下。
下同

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md [hunk 4]
```diff
@@ -547,6 +547,77 @@ CREATE CATALOG hive_krb_ha PROPERTIES (
...
+
+查找kdc时使用principal中的domain_name去找相对应的realm
+
+``` properties
+[libdefaults]
+  dns_lookup_realm = true
+  dns_lookup_kdc = true
+[domain_realm]
+  .your-host.example = REALM2.COM
+  your-host.example = REALM2.COM
```
Comment (morningman) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md:580]: 这里的示例两个是一样的？

Comment (wsjz) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/datalake-analytics/hive.md:580]: 这个是需要映射不同后缀

**Comment by wsjz** [comment]
run buildall

**Comment by wsjz** [comment]
> 1. Please also update the EN doc.
> 2. Is the doc also applicable to the 2.1 version? If so, update the 2.1 version docs.

done

**Comment by wsjz** [comment]
run build

**Review by luzhijing — APPROVED** [review]
LGTM

**Review by shasevendow — APPROVED** [review]

**Pull request merged**